### ISSUE TITLE
Automated Tagging not working with S3 as Primary Storage

### ISSUE THREAD

**Issue by LutzFinsterle** [nextcloud/files_automatedtagging#53]
### Steps to reproduce
1. Take Nextcloud instance with Primary S3
2. Create Workflow tagging every File with upload Size >= 1 B 
3. Upload File
4. No Tag ist assigned

### Expected behaviour
The Tag should be assigend to File

### Actual behaviour
No Tag is assigned

### Server configuration
**Operating system**:
centos7.4

**Web server:**
 Apache/2.4.6 (CentOS) OpenSSL/1.0.2k-fips PHP/7.2.5 configured 

**Database:**
Type: mysql
Version: 10.2.14
Size: 744.8 MB

**PHP version:**
Version: 7.2.5
Memory Limit: 512 MB
Max Execution Time: 3600
Upload max size: 511 MB

**Nextcloud version:** (see Nextcloud admin page)
13.0.2
**Where did you install Nextcloud from:**
Update from Site

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.

Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- core
	- EXTRA_FILE
		- password_reset.html

Raw output
==========
Array
(
    [core] => Array
        (
            [EXTRA_FILE] => Array
                (
                    [password_reset.html] => Array
                        (
                            [expected] => 
                            [current] => 5fcb6fbb0ea472c0a77018828bfd4b04e96fa061d1cc2feb851705786db069790f718bc485523306199ba2738663c5a5b6430872c52bafc222908b0ef057b6cf
                        )

                )

        )

)

```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
Enabled:
  - activity: 2.6.1
  - admin_audit: 1.3.0
  - bruteforcesettings: 1.0.3
  - circles: 0.13.6
  - comments: 1.3.0
  - dav: 1.4.6
  - federatedfilesharing: 1.3.1
  - federation: 1.3.0
  - files: 1.8.0
  - files_accesscontrol: 1.3.0
  - files_automatedtagging: 1.3.0
  - files_external: 1.4.1
  - files_pdfviewer: 1.2.1
  - files_sharing: 1.5.0
  - files_texteditor: 2.5.1
  - files_trashbin: 1.3.0
  - files_versions: 1.6.0
  - files_videoplayer: 1.2.0
  - firstrunwizard: 2.2.1
  - gallery: 18.0.0
  - limit_login_to_ip: 1.0.4
  - logreader: 2.0.0
  - lookup_server_connector: 1.1.0
  - nextcloud_announcements: 1.2.0
  - notifications: 2.1.2
  - oauth2: 1.1.0
  - password_policy: 1.3.0
  - provisioning_api: 1.3.0
  - serverinfo: 1.3.0
  - sharebymail: 1.3.0
  - survey_client: 1.1.0
  - systemtags: 1.3.0
  - theming: 1.4.4
  - twofactor_backupcodes: 1.2.3
  - updatenotification: 1.3.0
  - user_ldap: 1.3.1
  - user_usage_report: 1.0.1
  - workflowengine: 1.3.0
Disabled:
  - encryption
  - user_external

**Nextcloud configuration:**

{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "dest-docl03.itc.global.mahle",
            "dest-docllb1.itc.global.mahle",
            "intmcs.mahle.com"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "tempdirectory": "\/mnt\/test\/",
        "overwrite.cli.url": "https:\/\/dest-docl03.itc.global.mahle",
        "dbtype": "mysql",
        "version": "13.0.2.1",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "3306",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379,
            "timeout": 0
        },
        "log_type": "file",
        "loglevel": 0,
        "logfile": "\/nextcloud_data\/data\/nextcloud.log",
        "log_rotate_size": 104857600,
        "mail_smtpmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "25",
        "mail_smtpauthtype": "LOGIN",
        "objectstore": {
            "class": "OC\\Files\\ObjectStore\\S3",
            "arguments": {
                "bucket": "nextcloudcluster",
                "autocreate": true,
                "key": "",
                "secret": "",
                "hostname": "crgw.itc.global.mahle",
                "port": 443,
                "use_ssl": true,
                "region": "optional",
                "use_path_style": true
            }
        },
        "updater.release.channel": "stable",
        "maintenance": false,
        "ldapIgnoreNamingRules": false,
        "ldapProviderFactory": "\\OCA\\User_LDAP\\LDAPProviderFactory",
        "proxy": "dest-prxl00.itc.global.mahle:3128",
        "lost_password_link": "https:\/\/intmcs.mahle.com\/password_reset.html"
    }
}

**Are you using external storage, if yes which one:**  S3

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

### Client configuration
**Browser:**
firefox, Chrome, IE
**Operating system:**
Linux/Windows
### Logs
Nothing in the logs

**Comment by MorrisJobke** [comment]
Fix is in #54

**Comment by MorrisJobke** [comment]
This is tracked now in the server: https://github.com/nextcloud/server/issues/10109
